testsuite/azure-pipelines.yml

```diff
@@ -0,0 +1,38 @@
+trigger:
+- none
+  
+resources:
+- repo: self
+  
+stages:
+- stage: Build
+  displayName: Build image
+  jobs:  
+  - job: Build
+    displayName: Build
+    pool:
+      vmImage: 'ubuntu-latest'
+    steps:
+    - task: Docker@2
+      displayName: Build Conformance test launcher image
+      inputs:
+        containerRegistry: 'ConformanceACR'
+        repository: 'arck8sconformance/launcher'
+        command: build
+        Dockerfile: 'testsuite/Dockerfile'
+        buildContext: 'testsuite'
+        tags: $(launcherImageVersion)
+        arguments: '--build-arg SONOBUOY_VERSION=$(sonobuoyVersion) --build-arg AZURE_CLI_VERSION=$(azureCliVersion)'
+        addPipelineData: false
+    - task: Docker@2
+      displayName: Build and Push Conformance test launcher image
```

Comment:
Nit: It should be only Push & not 'Build & Push'

Reply:
ack